project-p2-layouts-master/code/style.css

```diff
@@ -0,0 +1,252 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;500;600;900&display=swap');
+
+body {
+  background-color:	rgb(177, 212, 195);
+}
+
+img {
+  width: 60%;
+  border-radius: 50%;
+}
+.center {
+  display: block;
+  margin-left:auto;
+  margin-right: auto; 
+  width: 50%; 
+}
+#mySidenav a {
+  position: absolute; 
+  left: -40px; 
+  transition: 0.3s; 
+  padding: 40px; 
+  margin: 50px; 
+  width: 50px; 
+  height: 50px;
+  text-decoration: none; 
+  font-size: 14px; 
+  font-family: 'Roboto Slab', serif ;
+  font-weight: bolder;
+  color: rgb(249, 250, 249); 
+  border-radius: 50%; 
+  text-align: center;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+}
+
+#mySidenav a:hover {
+  left: 0; 
+}
+#HOME {
+  top: 0px;
+  background-color:rgb(205, 241, 229);
+  margin-top: 120px; 
+  margin-bottom: 120px;
+}
+
+#IDEE {
+  top: 150px;
+  background-color: rgb(162, 216, 197);
+  margin-top: 120px;
+  margin-bottom: 120px;
+}
+
+#FRAGEBOGEN {
+  top: 300px;
+  background-color:rgb(140, 186, 172);
+  margin-top: 150px;
+  margin-bottom: 120px; 
+  margin-left: 2px;
+}
+
+#ÜBER {
```

Comment:
All code (including class names and id's) should be in English since that's the global programming language